packages/sync/src/modules/Posts.php

```diff
@@ -74,6 +74,17 @@ public function name() {
 		return 'posts';
 	}
 
+	/**
+	 * The table in the database.
+	 *
+	 * @access public
+	 *
+	 * @return string|bool
```

Comment:
I realize we've put the `bool` here because of the original implementation, but maybe it makes sense to leave it just `string` because this one will really always return a `string`. Maybe we should do it for the rest of the cases and leave it `string|bool` just in the abstract `Module` class? Or as I suggested in another comment, we can just declare the method abstract in the `Module` class, and have it always return a `string` that way. The bonus of that implementation is that we won't need the extra check for non-bool `$this->table_name()`, and the language will protect us from not implementing the method in the modules that inherit the abstract class.

```suggestion
	 * @return string
```